.kokoro/generate-docs.sh

```diff
@@ -74,7 +74,7 @@ for bucket_item in $(gsutil ls 'gs://docs-staging-v2/docfx-python*' | sort -u -t
   fi
 
   for tag in ${GITHUB_TAGS}; do
-    git checkout ${tag}
+    #git checkout ${tag}
```

Comment:
Seems unrelated?

Reply:
Ah yes, it was to test only against `HEAD` and not the latest release for some library version issues. I'll have that reverted.